pkg/controller/noobaa/noobaa_controller.go

```diff
@@ -57,6 +57,15 @@ func Add(mgr manager.Manager) error {
 	// Predicate that allow us to log event that are being queued
 	logEventsPredicate := util.LogEventsPredicate{}
 
+	// Predicate that filter events that noobaa is not their owner
+	filterForOwnerPredicate := util.FilterForOwner{
+		OwnerSubstrings: []string{"noobaa"},
+	}
+	// Predicate that filter pod events that one of noobaa statefulset/deployments is not their owner
+	podfilterForOwnerPredicate := util.FilterForOwner{
+		OwnerSubstrings: []string{"noobaa-operator", "noobaa-core", "noobaa-db", "noobaa-endpoint"},
+	}
```

Comment:
Is there a benefit of using a shared instance of the predicate, it seems that each resource type has a diff Owner and I think that it will be more precise to use an inline instantiation of the predicate with the proper owner on each watch

Reply:
deleted it according to our conversation so it's irelevant.